torchgeo/datamodules/oscd.py

```diff
@@ -86,9 +86,11 @@ def __init__(
         self.std = torch.tensor([STD[b] for b in self.bands])
 
         self.aug = AugmentationSequential(
-            K.Normalize(mean=self.mean, std=self.std),
-            _RandomNCrop(self.patch_size, batch_size),
-            data_keys=['image1', 'image2', 'mask'],
+            K.VideoSequential(
+                K.Normalize(mean=self.mean, std=self.std),
+                _RandomNCrop(self.patch_size, batch_size),
+            ),
+            data_keys=['image', 'mask'],
```

Comment:
Shouldn't this be `data_keys=None` so that Kornia automatically detects the key name from the dict keys?

Reply:
Yes, I updated this when resolving the merge conflict.